sdcm/data_operations/data_operation.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+import logging
+import random
+import threading
+import time
+from functools import cached_property
+
+from sdcm.utils.user_profile import get_profile_content, get_table_by_stress_cmd, get_keyspace_by_stress_cmd
+
+
+class ResurrectionDataFound(Exception):
+    pass
+
+
+class SchemaManage:
+    """
+    Manage the test schema.
```

Comment:
this class is doing is mostly a collection of helper functions that seems a bit unrelated to each other

running a c-s, while create keyspaces or query name of views, almost non of those "manage" the schema.

Reply:
changed name to `SchemaHelpers`